aiotruenas_client/websockets/dataset.py

```diff
@@ -27,7 +27,9 @@ def available_bytes(self) -> int:
     @property
     def comments(self) -> Optional[DatasetProperty]:
         """The user-provided comments on the dataset."""
-        return self._get_property("comments")
+        property = self._get_property("comments")
+        assert property is not None
+        return property.parsedValue
```

Comment:
This no longer matches the type above (I think it should be `string`).  IIRC, the reason I had this return the full, optional, `DatasetProperty` was because it seemed like different usages might behave differently.